client/my-sites/stats/style.scss

```diff
@@ -18,6 +18,14 @@
 	margin-left: 4px;
 }
 
+.is-section-stats .followers-count .button {
```

Comment:
@mtias recently added a [style.scss](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/stats-navigation/style.scss) for `StatsNavigation`, perhaps we should put this code in there?